build/patches/Enable-crash-reporting-ui.patch

```diff
@@ -0,0 +1,872 @@
+--- a/components/crash/core/browser/resources/crashes.html
++++ b/components/crash/core/browser/resources/crashes.html
+@@ -1,6 +1,7 @@
+ <!doctype html>
+ <html dir="$i18n{textdirection}" lang="$i18n{language}">
+ <head>
++  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <meta charset="utf-8">
+ 
+ <if expr="is_ios">
+@@ -45,6 +46,21 @@
+         <input type="checkbox" id="showDevDetails">
+         $i18n{showDeveloperDetails}
+       </label>
++      <div>
++        <button class="button" id="clearAll">
++          $i18n{clearAll}
++        </button>
++        <button class="button" id="newExtraction">
++          $i18n{extractNow}
++        </button>
++        <div class="spinner" id="spinner" hidden>
++          <div class="rect1"></div>
++          <div class="rect2"></div>
++          <div class="rect3"></div>
++          <div class="rect4"></div>
++          <div class="rect5"></div>
++        </div>
++    </div>
+     </h2>
+ 
+     <div id="crashList">
+diff --git a/components/crash/core/browser/resources/crashes.js b/components/crash/core/browser/resources/crashes.js
+--- a/components/crash/core/browser/resources/crashes.js
++++ b/components/crash/core/browser/resources/crashes.js
+@@ -32,6 +32,7 @@ function updateCrashList(
+ 
+   $('disabledMode').hidden = enabled;
+   $('crashUploadStatus').hidden = !enabled || !dynamicBackend;
++  $('spinner').hidden = true;
+ 
+   const template = crashList.getElementsByTagName('template')[0];
+ 
+@@ -98,22 +99,16 @@ function updateCrashList(
+ 
+       uploadTime.querySelector('.value').textContent = crash.upload_time;
+ 
+-      sendNowButton.remove();
+-      fileBugButton.onclick = () => fileBug(crash.id, os, version);
++      fileBugButton.remove();
+     } else {
+       uploadId.remove();
+       uploadTime.remove();
+       fileBugButton.remove();
+-      // Do not allow crash submission if the Chromium build does not support
+-      // it, or if the user already requested it.
+-      if (!manualUploads || crash.state === 'pending_user_requested') {
+-        sendNowButton.remove();
+-      }
+-      sendNowButton.onclick = (e) => {
+-        e.target.disabled = true;
+-        chrome.send('requestSingleCrashUpload', [crash.local_id]);
+-      };
+     }
++    sendNowButton.onclick = (e) => {
++      e.target.disabled = true;
++      chrome.send('requestSingleCrashUpload', [crash.local_id]);
++    };
+ 
+     const fileSize = crashRow.querySelector('.fileSize');
+     if (crash.file_size === '') {
+@@ -184,6 +179,27 @@ function requestCrashUpload() {
+   refreshCrashListId = setTimeout(requestCrashes, 5000);
+ }
+ 
++/**
++ * Request new log extraction.
++ */
++ function requestNewExtraction() {
++  chrome.send('requestNewExtraction');
++
++  // show spinner
++  $('spinner').hidden = false;
++
++  // Trigger a refresh in 3 seconds.  Clear any previous requests.
++  clearTimeout(refreshCrashListId);
++  refreshCrashListId = setTimeout(requestCrashes, 3000);
```

Comment:
How come here it's 3 seconds and in `requestCrashUpload` it's 5 seconds?

Reply:
no motivation, is that in my tests 5 seconds seemed too much!